trade-finance-manager-ui/templates/utilisation-reports/_macros/payment-details-filters-panel.njk

```diff
@@ -8,79 +8,86 @@
   {% set errors = params.errors %}
   {% set reportId = params.reportId %}
 
-  <div class="moj-filter-layout__filter" data-filtered-table-id="payment-details" data-cy="payment-details--filters-panel">
+  {% if object.field2 %}
```

Comment:
what's this and the field3 stuff below?

Reply:
Ah whoops, that was left in by accident!